server/build.gradle

```diff
@@ -3,7 +3,9 @@ description = 'CloudFoundry Identity Server JAR'
 dependencies {
 
   compile project(':cloudfoundry-identity-model')
-
+  compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version:parent.tomcatVersion
+  compileOnly group: 'org.projectlombok', name: 'lombok', version: parent.lombokVersion
```

Comment:
Please check whether you still need `compileOnly` if you already have `annotationProcessor` on the same dependency.

Reply:
I've checked. It will not compile without the "compileOnly" line.